lib/tasks/start_coordinates.rake

```diff
@@ -0,0 +1,14 @@
+desc "Populate start coordinates of stream with data from first measurement"
+task :populate_start_coordinates => [:environment] do
+  Stream.all.each do |stream|
+    first_measurement = stream.measurements.order(time: :asc).first
+    if first_measurement
+      puts stream.id
```

Comment:
If `Stream.all` returns in a different order than id ascending I'm not sure if this yields valuable info

Reply:
It's fine, I just wanted to see the progress, but instead of dots I used id, cause I also wanted to see if there are any streams without measurements.